gobblin-metastore/src/main/java/gobblin/metastore/FsStateStore.java

```diff
@@ -281,7 +280,7 @@ public void createAlias(String storeName, String original, String alias) throws
     Path aliasTablePath = new Path(new Path(this.storeRootDir, storeName), alias);
     // Make a copy of the original table as a work-around because
     // Hadoop version 1.2.1 has no support for symlink yet.
-    FileUtil.copy(this.fs, originalTablePath, this.fs, aliasTablePath, false, true, this.conf);
+    HadoopUtils.copyPath(this.fs, originalTablePath, this.fs, aliasTablePath, this.conf);
```

Comment:
Since `HadoopUtils.copyPath` can no longer overwrite, you need to first delete `current.jst`. However, if `current.jst` is deleted and then `copyPath` failed, there will be no `current.jst` and the next run will have no state.